src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -9,5 +9,6 @@ public class Messages {
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_NRIC_NOT_EXIST = "There is no such person with the given NRIC!";
```

Comment:
Should we put this in deleteElderlyCommand, similar to the other  MESSAGE_DUPLICATE_PERSON in addElderlyCommand?

Reply:
Actually, I think this is better since we're planning to put `Nric` in `Person`. If not, there'll be a lot of duplicate message. Instead, 
```suggestion
    public static final String MESSAGE_NRIC_NOT_EXIST = "There is no such %s with the given NRIC!";
```
What do you think of this?

Reply:
not good 👎